app/components/subcases/subcase-header.js

```diff
@@ -84,6 +84,10 @@ export default class SubcasesSubcaseHeaderComponent extends Component {
     this.subcaseToDelete = subcase;
   }
 
+  get hasActions() {
+    return this.canDelete || this.canPropose || this.args.parliamentFlow;
```

Comment:
@SchrodingersCat00 here as well, right? Should be `!this.args.parliamentFlow` I think.

Reply:
You're right, not sure how I missed that